pkg/ip/link_linux_test.go

```diff
@@ -233,6 +233,37 @@ var _ = Describe("Link", func() {
 			})
 		})
 
+		It("successfully creates a veth pair with an explicit mac", func() {
+			const mac = "02:00:00:00:01:23"
```

Comment:
why this test case is separated into three steps? and I don't see the necessity of the third step